src/managers/GuildMemberManager.js

```diff
@@ -223,6 +223,25 @@ class GuildMemberManager extends CachedManager {
     return data.reduce((col, member) => col.set(member.user.id, this._add(member, cache)), new Collection());
   }
 
+  /**
+   * Data that can be resolved to a Date object for timeouts. This can be:
+   * * A number in seconds
+   * @typedef {DateResolvable} TimeoutDateResolvable
```

Comment:
The way the code is set atm this accepts a date object, as well as an ISO8601 string (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date)
```suggestion
   * Data that can be resolved into a Date object for timeouts. This can be:
   * * A number in seconds
   * * An {@link https://en.wikipedia.org/wiki/ISO_8601 ISO 8601} string
   * * A {@link Date} object
   * @typedef {DateResolvable} TimeoutDateResolvable
```

Reply:
That's exactly what this type definition explains. What you have done is repeat ISO 8601 and the date object twice.

Reply:
Well then this should still be fixed, but disregard the previous suggestion (yes this is done elsewhere, see https://discord.js.org/#/docs/main/stable/typedef/EmojiIdentifierResolvable for example)
```suggestion
   * Data that can be resolved to a Date object for timeouts. This can be:
   * * A number in seconds
   * * A DateResolvable
   * @typedef {DateResolvable} TimeoutDateResolvable
```